### PR TITLE
Fix ls-tree parsing bug and use dist: trusty for Travis

### PR DESCRIPTION
**Pull request by saurvs**
Output format of `git ls-tree` with the `-l` option is

```
<mode> SP <type> SP <object> SP <object size> TAB <file>
```

Currently, if a filename begins with whitespace, Gitter will strip that whitespace. This PR fixes the splitting logic to consider the `TAB` as well and return the correct filename.

### CODE REVIEW

**Comment by saurvs** [comment]
@klaussilveira Could you please take a look as to why the Travis builds for PHP 5.4 and 5.5 have errored before PHPUnit tests are run?

File: lib/Gitter/Model/Tree.php [hunk 1]
```diff
@@ -39,7 +39,10 @@ public function parse()
                 continue;
             }
 
-            $files[] = preg_split("/[\s]+/", $line, 5);
+            $tab_split = preg_split("/[\t]+/", $line, 2);
```
Comment (klaussilveira) [inline lib/Gitter/Model/Tree.php]: Can you switch to `camelCase`?

**Comment by klaussilveira** [comment]
https://github.com/travis-ci/travis-ci/issues/7014

File: lib/Gitter/Model/Tree.php (hunk 1 above)
Comment (saurvs) [reply lib/Gitter/Model/Tree.php]: done

**Comment by saurvs** [comment]
@klaussilveira Based on this thread https://travis-ci.community/t/php-5-4-and-5-5-archives-missing/3723/2, using `dist: trusty` worked.

**Comment by saurvs** [comment]
@klaussilveira Is it possible to release a new Gitter version for this change? We need it for Gitlist.

**Comment by klaussilveira** [comment]
Done!

**Comment by saurvs** [comment]
Thank you @klaussilveira !

**Pull request merged**